build.gradle

```diff
@@ -182,6 +194,7 @@ allprojects {
           // ZooKeeper (potentially older and containing CVEs)
           libs.nettyHandler,
           libs.nettyTransportNativeEpoll,
+          libs.slf4jApi,
```

Comment:
why we need this change?

Reply:
```
spotbugsSlf4j - configuration for the SLF4J provider to run SpotBugs
\--- org.slf4j:slf4j-simple:2.0.0
     \--- org.slf4j:slf4j-api:2.0.0
```

spotbugs requires the 2.x slf4j, so forcing the version can cause log issue in running spotbugs

Reply:
I previously encountered version inconsistency issues on my local machine. While those issues have been resolved, this change represents a modification that should have been removed but was accidentally missed during cleanup. 😬